rllib/utils/nested_dict.py

```diff
@@ -72,7 +72,7 @@ class NestedDict(Generic[T], MutableMapping[str, Union[T, "NestedDict"]]):
         x: a representation of a nested dict: it can be an iterable of `SeqStrType`
         to values. e.g. `[(("a", "b") , 1), ("b", 2)]` or a mapping of flattened
         keys to values. e.g. `{("a", "b"): 1, ("b",): 2}` or any nested mapping,
-        e.g. `{"a": {"b": 1}, "b": 2}`.
+        e.g. `{"a": {"b": 1}, "b": {}}`.
```

Comment:
@kouroshHakha Do you know all the places where we state or make this assumption that there are no empty branches in the nested dict? I could not find anything in this file.

Reply:
No I think it was more like an implicit assumption to make iteration easier.